Department.php

```diff
@@ -0,0 +1,30 @@
+<?php
+    class Department
+    {
+        public function __construct(private string $name, private $Employees)
+        {
+        }
+        public function get_name(): string{
+            return $this->name;
+        }
```

Comment:
Между методами должна быть пустая строка
```suggestion
        }
        
```